README.md

```diff
@@ -392,7 +394,3 @@ No additional parameters required
 ### Get Tags
 
 `GET /api/tags`
```

Comment:
@kdaglas Add a newline at the end of this file.

Reply:
I have added it